test/javascript/tests/view_errors.js

```diff
@@ -154,7 +154,7 @@ couchTests.view_errors = function(debug) {
           db.view("infinite/infinite_loop");
           T(0 == 1);
       } catch(e) {
-          T(e.error == "os_process_error");
+
```

Comment:
Why was this removed? If this test passes then we should be asserting that we have the expected error here.

Reply:
there is different error generated for spidermonkey 6.0. I added both for that now https://github.com/apache/couchdb/pull/2345/commits/9d1cb180760cedcc2ec74faafb3298ee1af4f494#diff-4aba4f69c2f30c9686f1c16ea474a9ddR157